src/app/modules/item/pages/item-display/item-display.component.html

```diff
@@ -28,16 +28,18 @@
     </ng-container>
 
     <div class="error-buttons">
-      <alg-button
+      <p-button
         class="error-button"
+        styleClass="alg-button"
         i18n-label label="Show task anyway (for debugging)"
         (click)="showTaskAnyway = true"
-      ></alg-button>
-      <alg-button
+      ></p-button>
+      <p-button
         class="error-button"
+        styleClass="alg-button"
         i18n-label label="Retry"
         (click)="refresh.emit()"
```

Comment:
I'm confused, everywhere else you changed `(click)` to (onClick)` but not here and it still works, so both `(click)` and `(onClick)` are ok to use with p-button?

Reply:
(click) doesn't block event when disabled state